solidity/contracts/Peggy.sol

```diff
@@ -126,10 +128,10 @@ contract Peggy {
 		// This is what we are checking they have signed
 		bytes32 _theHash,
 		uint256 _powerThreshold
-	) private pure returns (bool) {
```

Comment:
maybe I just need to read the audit, but why is returning a boolean here bad? Because it's not required thanks to the throw?

Reply:
yep